spec/controllers/chapters/changes_controller_spec.rb

```diff
@@ -28,10 +28,10 @@
   end
 
   describe 'chapter is not valid at given date', vcr: { cassette_name: 'chapters_changes#index_0100000000_1970-01-01' } do
-    before do
+    let :request_page do
       get :index, params: { chapter_id: chapter.short_code, as_of: Date.new(1970, 1, 1) }, format: :atom
     end
 
-    it { is_expected.to respond_with(:not_found) }
```

Comment:
I think it does kind of make sense to have an expectation on the response in the controller, it's like a small integration test, that tests the ExceptionWrapper too.

Reply:
Yeh - I dropped this because it wont work (the request object gets reused across the rest of the specs, which means it wont pass without turning off 'normal' exception handling in specs for the whole test environment :disappointed: I'd had an unsuccessful attempt at enabling that just for these specs, I may retry